lib/section-to-webhook.ts

```diff
@@ -51,17 +52,40 @@ const getDescription = ($: cheerio.Root): string =>
     .map((html) => turndownService.turndown(html))
     .join("\n\n");
 
-const getActions = ($: cheerio.Root): string[] => {
-  const unwantedActions = ["true", "false", "status"];
-
+const getPropertiesEl = ($: cheerio.Root) => {
   const $table = $('[id^="webhook-payload-object"]').next("table");
 
   if (!$table.is("table")) {
     return [];
   }
 
-  const [keyEl, , descriptionEl] = $table
-    .find("tbody tr:first-child td")
+  return $table.find("tbody tr").get() as cheerio.Element[][];
+};
+
+const getProperties = ($: cheerio.Root) => {
+  const propertiesEl = getPropertiesEl($).slice(1);
```

Comment:
```suggestion
  const [, ...propertiesEl] = getPropertiesEl($);
```

nit: I don't mind using `slice`, but find using `...` a bit nicer in these situations as you don't have to think about the behaviour of `slice` (i.e "is this taking the first element, or all elements except the first 🤔")